src/libs/s3-utils.ts

```diff
@@ -0,0 +1,54 @@
+import { S3 } from "aws-sdk"
+
+/**
+ * Set of S3 utilities
+ */
+namespace S3Utils {
+
+    /**
+     * Loads JSON from S3
+     * 
+     * @param s3 S3 client
+     * @param key object key
+     * @returns object data as JSON or null if not found
+     */
+    export const loadJson = async <T> (s3: S3, bucket: string, key: string): Promise<T | null> => {
+        try {
+            const object = await s3.getObject({
+                Bucket: bucket,
+                Key: key
+            }).promise();
+        
+            const body = object.Body;
+            if (!body) {
+                return null;
+            }
+        
+            return JSON.parse(body.toString());
+        } catch (e: any) {
+            if (e.code === "NoSuchKey") {
+                return null;
+            }
+            
+            throw e;
+        }
```

Comment:
This looks a little suspicious but I'm happy if you are, is this how it was in the previous code?

Reply:
This code was taken from the old calendar (on-call-ui), with minor modifications by Antti to how the files are fetched from the bucket, as that was done differently in the SST framework. I'll leave it as it is for now.